(DOCS-3830) Getting started for MSPs (#14964)
DataDog/documentation: content/en/partners/delivering-value.md

- [Monitor settings changes not taking effect][18]

### Downtimes

A common problem with monitors and alerts is alert fatigue, where an overabundance of alarms or notifications causes desensitization to the alarms. One way to combat alert fatigue is to limit the number of false positive alarms. This is especially pertinent in controlled situations such as planned system shutdowns, maintenance, or upgrade-windows.

Datadog's Downtimes offer you and your clients a way to mute monitors during times of planned (or ad hoc) maintenance.

To find out more about managing downtimes, especially programmatically, see:
- [Downtime][19]
- [Mute Datadog Alerts for Planned Downtimes][20] (Blog)
- [Managing Datadog with Terraform][21] (Blog)
- [Downtime API][22]
- [Prevent alerts from Monitors that were in downtime][23]

### Notifications

Some general guidelines for notifications:
- Alert liberally; page judiciously
- Page on symptoms, rather than causes

Datadog offers various channels through which you or your clients can notify users about important alerts:

- Email notifications
- Integrations, such as:
  - [Slack][24]
  - [PagerDuty][25]
  - [Flowdock][26]
  - [ServiceNow][27]

>   - [Google Hangouts Chat][28]

Reviewer: ```suggestion
  - [Google Chat][28]
```

rip hangouts

  - [Microsoft Teams][29]
  - And [many more][19]

> You can also invoke any REST API using the generic [Webhooks integration][30]. You use a Webhooks integration to not only notify users but also to trigger automatic remediation workflows.

Reviewer: ```suggestion
You can also invoke any REST API using the generic [Webhooks integration][30]. You use a Webhooks integration to not only notify users, but also to trigger automatic remediation workflows.
```

To find out more about notifications, see:
- [Notifications][31]
- [Send SMS Alerts with Webhooks and Twilio][32] (Blog)

> ## Setting up visualizations with Dashboards

Reviewer: ```suggestion
## Setting up visualizations with dashboards
```

Here are some dashboard best practices to take into consideration when building your dashboards:
- Focus on work metrics rather than too many resource metrics. To understand the difference, see [Monitoring 101: Collecting the right data][33] (blog).
- Make use of [event overlays][34] to correlate metrics and events.
- Annotate the dashboards with [free text information][35] on what data is shown and what to do in case the dashboard indicates an issue.

To find out more about Dashboards, see:
- [Building Better Dashboards][36] (Training)
- Use the [Datadog Notebooks][37] feature to gather data in an exploratory fashion and draft dashboards
- [Monitor Datacenters and Networks with Datadog][38] (Blog)
- [Use Associated Template Variables][39] (Blog)
- [The Datadog Dashboard API][40]
- [Configure Observability as Code with Terraform & Datadog][41] (HashiCorp webinar)

### Visualizations for users without Datadog access

Depending on your business model, your clients might not require access to Datadog themselves. However, you might still want to provide Datadog visualizations to your clients. You have the following options for providing Datadog visualizations:

> - [Dashboard sharing][42]: Provide a status page to your clients by sharing a public URL to a read-only dashboard, or share the dashboard privately using an individual e-mail address.

Reviewer: ```suggestion
- [Dashboard sharing][42]: Provide a status page to your clients by sharing a public URL to a read-only dashboard, or share the dashboard privately using an individual email address.
```
